ilc/server/errorHandler/ErrorHandler.spec.js

```diff
@@ -30,7 +36,7 @@ describe('ErrorHandler', () => {
 
     it('should show 500 error page with an error id', async () => {
         nock(config.get('registry').address).get('/api/v1/router_domains').reply(200, []);
-        nock(config.get('registry').address).get(`/api/v1/template/500/rendered?locale=en-US`).reply(200, {
+        nock(config.get('registry').address).get(`/api/v1/template/500/rendered?locale=en-US&domain=${address}`).reply(200, {
```

Comment:
you updated existed test case with an additional query `domain`. but should we create another test case to have different tests cases such as `with domain query` and `without domain query`?